components/SearchBar.vue

```diff
@@ -79,6 +80,28 @@ const selectedSpecialty: Ref<Specialty | String> = ref('')
 const selectedLocation: Ref<string> = ref('')
 const selectedLanguage: Ref<Locale | String> = ref('')
 
+const anyLocation = "----Any----"
```

Comment:
like the variable, but the name wasn't clear to me at first what it was.  (I read if the city is literally any location, then return)

```suggestion
const placeholderDisplayText = "----Any----"
```